src/Engine/GamePath.java

```diff
@@ -19,16 +19,30 @@ public class GamePath
 
   public GamePath()
   {
-    waypoints = new ArrayList<PathNode>();
+    waypoints = new ArrayList<>();
+  }
+  public static GamePath getStatic(Unit unit)
+  {
+    return getStatic(new XYCoord(unit));
+  }
+  public static GamePath getStatic(XYCoord xyc)
```

Comment:
I might call this `stayPut` or `to[Now]here` or 'dummy` or something just because "static" can mean things besides "stationary", especially in a programming context.

Reply:
I was coming up with `getNull`, which doesn't really work.
I like `getDummy` better, but `stayPut` is real nice.

Reply:
>> I like `getDummy` better
>
> Implements `stayPut`

-\\_(^ ^)_/-

Reply:
The referent of "better" was my existing solution